### DISCUSSION TITLE
Had my first print failure on a long print (12h) (extruder stopped extruding near end). Whats considered a safe length of time using this firmware? 6h prints have been fine for me

### DISCUSSION THREAD

**Discussion by fernplant** [xblax/flashforge_ad5m_klipper_mod#269] [General]
Had my first print failure on a long print (12h) (extruder stopped extruding near end). Whats considered a safe length of time using this firmware? 6h prints have been fine for me

**Comment by MACCAMENZIES** [comment]
Never had an issue with this firmware. Its more reliable than the stock Flashforge Firmware for sure. Flashforge are not interested in fixing the factory firmware. At least with this community you can always get answers or help to any issues you might have. Been running V5 for months.
